app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -0,0 +1,115 @@
+import { TooltipMenu } from 'components/tooltip-menu';
+import React from 'react';
+import { FaFileExport, FaPlay } from 'react-icons/fa6';
+import { useReports } from 'store/hooks';
+import { useProfileStore } from 'store/slices';
+import { IContentModel, IReportInstanceContentModel, IReportModel, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IAddToReportMenuProps {
+  content: IContentModel[];
+}
+
+export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content }) => {
+  const [{ updateReport, findMyReports }] = useReports();
+  const [{ myReports }] = useProfileStore();
+  const [activeReport, setActiveReport] = React.useState<IReportModel | null>(null);
+  React.useEffect(() => {
+    if (!myReports.length) {
+      findMyReports().catch(() => {});
+    }
+    // Only do this on init.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  /** Adds the content to the active report. */
```

Comment:
I feel like this isn't the correct way... let me know. I essentially look for the latest instance and add the content to it with the existing content that is there

Reply:
Updating now to handle case where no instances are present